conf/bafu.conf.part

```diff
@@ -1090,7 +1090,7 @@ source src_ch_bafu_wasser_vorfluter : def_searchable_features
         SELECT bgdi_id as id \
             , id as label \
             , 'feature' as origin \
-            , remove_accents(coalesce(id, ' ')||' '||coalesce(ezgnr, ' ')||' '||coalesce(gewaessername, ' ')||' '||coalesce(gwlnr, ' ')) as  detail \
+            , remove_accents(coalesce(id::text, ' ')||' '||coalesce(ezgnr::text, ' ')||' '||coalesce(gewaessername, ' ')||' '||coalesce(gwlnr, ' ')) as  detail \
```

Comment:
```SQL
concat_ws(' ', id, ezgnr, gewaessername, gwlnr)
```
is the best way to concatenate text, it can handle different input data types without manual casting.